test/e2e/tls_test.go

```diff
@@ -99,13 +102,13 @@ func (data *TestData) configureTLS(t *testing.T, cipherSuites []uint16, tlsMinVe
 	}
 }
 
-func (data *TestData) checkTLS(t *testing.T, podName string, containerName string, apiserver int, apiserverStr string) {
+func (data *TestData) checkTLS(t *testing.T, podName string, containerName string, apiserver int, apiserverStr string, nodeIPv4, nodeIPv6 string) {
```

Comment:
in the context of this function, it makes more sense to call the parameters `dstIPv4` and `dstIPv6` instead of `nodeIPv4` and `nodeIPv6`